core/src/test/java/com/linecorp/armeria/client/proxy/ProxyClientIntegrationTest.java

```diff
@@ -546,6 +544,17 @@ protected void channelRead0(ChannelHandlerContext ctx, FullHttpRequest msg) thro
         }
     }
 
+    private static final class SleepHandler extends ChannelInboundHandlerAdapter {
+
+        @Override
+        public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exception {
+            if (evt instanceof ProxySuccessEvent) {
+                Thread.sleep(Flags.defaultWriteTimeoutMillis());
```

Comment:
Could you add some comments about what this sentence does? e.g. why we block the event loop

Reply:
Fixed. 😉